test/extensions/quic_listeners/quiche/envoy_quic_proof_source_test.cc

```diff
@@ -43,6 +43,7 @@ class TestGetProofCallback : public quic::ProofSource::Callback {
         .WillByDefault(ReturnRef(empty_string));
     ON_CALL(client_context_config_, certificateValidationContext())
         .WillByDefault(Return(&cert_validation_ctx_config_));
+    ON_CALL(client_context_config_, sslctxCb()).WillByDefault(Return([](SSL_CTX*) {}));
```

Comment:
Is this needed? Won't the default return nullptr (or it's equivalent)?

Reply:
Done.